### ISSUE TITLE
Giant Jump Height.

### ISSUE THREAD

**Issue by Snacj** [FlamingTaco113/origins-datapack#5]
Hi, always searched for a Server Side only option of the Origins Mod. Big props from my side!
I want to create a new Orgin Inchling myself to add to your datapack. Only problem i face is the Jump height. I want to reduce the jump hight. Can you maybe explain how the jump height works with the Giant-Origin? I dont really understand that based of the code. You are using custom attributes right? 
Dont really know where you get the 'giant_jump' and 'add_multiplied_base' from in this line

attribute @s minecraft:generic.jump_strength modifier add giant_jump 1 add_multiplied_base

Thank you so much and keep going with the good work!

**Comment by FlamingTaco113** [comment]
Would you be adding that to the base Origins Datapack, or would this just be for a personal project/seperate datapack addon? 

Just to clarify, I am completely okay with you modifying my datapack for personal use, just as long as you do not redistribute it.

**Comment by Snacj** [comment]
Im trying to use it for me personally im playing on my own server with 1 or  2 friends

**Comment by FlamingTaco113** [comment]
Perfect!

For the jump attribute modifier, the giant_jump is used in to identify what the attribute is, so that if you reset the origin to change, it can be removed easily.

For inchling, if you'd want one block jump height for example, you would use
```
attribute @s minecraft:generic.jump_strength modifier add inchling_jump -0.14 add_value
``` 
This will make you jump roughly over half a block. You can play with the value to get a jump height you desire.

To remove the less jump height, all you need to use is
```
attribute @s minecraft:generic.jump_strength modifier remove inchling_jump
```

To reiterate, 
the "inchling_jump" is just a way to name the attribute you are giving the player, which is why
```
attribute @s minecraft:generic.jump_strength modifier remove inchling_jump
```
removes it.

Let me know if you have any more questions.

**Comment by Snacj** [comment]
Wow! Thank you so much for the detailed and fast explanation! Once again really love the datapack and your work! 
Best regards!

**Comment by FlamingTaco113** [comment]
If you have the time, would you mind sharing the process of how you added your custom origin? I want to make a template where people could make simple custom addon origins to the game, and am just curious as to how you went about doing it.

Thank you!

**Comment by Snacj** [comment]
I just picked one Origin (Dwarf) and went through each folder. Picked out the Origin and copied the files then modified them. Then I went through all the files and added the lines everywhere I found a mention of the Origin. Optionally if i needed move speed (for example) i looked at the avian because i knew it has a move speed modifier and copied it to my new Origin and modified the values.

I really just start to get to know datapacks so i wont know how to create custom attributes. A Template would probably need some kind of selection of attributes maybe or example attributes?

There is probably a way to use some kind of Data generation to automatically create entries in some files like:
choose_origin_load
check_picked
disable_picks
